beat/Functions/kinematic_routine_walking.m

```diff
@@ -95,51 +95,52 @@
 [aaa, name, extension]=fileparts(PlatformData);
 name2=regexprep(name,'_PlatformData','');
 file_id=fopen(strcat(outFolder,"/", name2, "_mROM", ".yaml"),'w'); %%open file to write into
-fprintf(file_id, "type: 'vector'\n");
-fprintf(file_id, "'measure unit: %%' \n");
-label_str="angle_label: [";
+fprintf(file_id, "type: 'labelled_matrix'\n");
+fprintf(file_id, "measure unit: '%%'\n");

```

Comment:
It is wrong also in the previous version, it must be degree °

Reply:
SI would be better, i.e. radians. But let's leave it as is for later on.

Reply:
When you are looking at human angles, it is usually expressed in ° (99.9% of literature). However, if you want we can express it in radians